internal/sdkv2provider/resource_cloudflare_workers_script.go

```diff
@@ -157,10 +163,15 @@ func resourceCloudflareWorkerScriptCreate(ctx context.Context, d *schema.Resourc
 
 	parseWorkerBindings(d, bindings)
 
+	compatibilityFlags := make([]string, 0)
+
+	getCompatibilityFlags(d, compatibilityFlags)
+
 	_, err = client.UploadWorker(ctx, cloudflare.AccountIdentifier(accountID), cloudflare.CreateWorkerParams{
 		ScriptName:        scriptData.Params.ScriptName,
 		Script:            scriptBody,
 		CompatibilityDate: d.Get("compatibility_date").(string),
+		CompatibilityFlags: compatibilityFlags,
```

Comment:
if you take my advice from above, you can remove the `compatibilityFlags` definition and reference the method call here.

```suggestion
		CompatibilityFlags: getCompatibilityFlags(d),
```

Reply:
(alternatively, you can follow the existing pattern of doing it all in the method and defining a variable to use 🤷 )